### PR TITLE
Beegions, But With Actual Bees

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
what if legions could spit bees
## Why It's Good For The Game
bad meme
## Changelog
:cl:
add: Beegions! Like Legions, but with actual bees. As in, the bees from the holodeck sim with the randomly-generated toxic bees.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 1]
```diff
@@ -135,6 +135,11 @@
 	crusher_drop_mod = 20
 	dwarf_mob = TRUE
 
+/mob/living/simple_animal/hostile/asteroid/hivelord/legion/beegion
+	name = "beegion"
+	desc = "You can still see what was once a human under the shifting mass of - oh my God, those are bees."
+	brood_type = /mob/living/simple_animal/hostile/poison/bees/toxin
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: Bees would attack the legion before the miner here.
Maybe make the bees be lava land themed and have all the normal protection a lava land mob gets.

Comment (Trilbyspaceclone) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: Nvm Im bad at reading vars

**Comment by IronEleven** [comment]
Needs to spawn if you say beegion five times in front of a mirror.

**Comment by RogueSteampunker** [comment]
We are beegion. We hate you.

**Comment by deathride58** [comment]
oh god, not the bees, NOT THE BEES!

**Review by deathride58 — APPROVED** [review]

**Pull request merged**